edit/edit.css

```diff
@@ -716,20 +774,27 @@ html:not(.usercss) .usercss-only,
 }
 
 /************ reponsive layouts ************/
-@media(max-width: 850px) {
+@media(max-width: 800px) {
```

Comment:
why was it changed?

Reply:
Combined 2 or 3 screens into 1 in some cases. Best estimate of when compact becomes necessary, but we can switch back to 850px.

Reply:
My point is, if there was no problem with the existing number then there's no need to change it. Same for the other screens.

Reply:
Sure, NBD.